### ISSUE TITLE
Decide which DB to use after all

### ISSUE THREAD

**Issue by matcool** [geode-sdk/server#4]
sqlite or postgres.. we have to choose

**Comment by FireMario211** [comment]
- sqlite is not good for long term (a lot of data being added over time), postgres is good for long term.
- sqlite doesn't require setting up anything besides the sql file (serverless), postgres requires setting up the db and server
- postgres is good for complex queries/fields while sqlite aims for simplicity, although I'm unsure what the final schema of the DB would be for the index.

I highly doubt we're going to have like 1 million mods, but either way both are good.

tldr; sqlite for simplicity and ease of use, postgres for complex (and scales well)

**Comment by dankmeme01** [comment]
postgres imo

**Comment by matcool** [comment]
postgres it is

**Comment by ZhulinskiiDanil** [comment]
postgres
